src/SharpGLTF.Toolkit/Geometry/Packed/PackedPrimitiveBuilder.cs

```diff
@@ -93,32 +93,57 @@ public void SetMorphTargets(IPrimitiveReader<TMaterial> srcPrim, PackedEncoding
             bool hasNormals = _VertexAccessors.Any(item => item.Attribute.Name == "NORMAL");
             bool hasTangents = _VertexAccessors.Any(item => item.Attribute.Name == "TANGENT");
 
+            bool hasColors0 = _VertexAccessors.Any(item => item.Attribute.Name == "COLOR_0");
+            bool hasColors1 = _VertexAccessors.Any(item => item.Attribute.Name == "COLOR_1");
+            bool hasTextCoords0 = _VertexAccessors.Any(item => item.Attribute.Name == "TEXCOORD_0");
+            bool hasTextCoords1 = _VertexAccessors.Any(item => item.Attribute.Name == "TEXCOORD_1");
+
             if (!hasPositions) throw new InvalidOperationException("Set vertices before morph targets.");

```

Comment:
Originally, this was required because unless you have positional morphing, it made no sense to have morphing at all, but now you can have material morphing without positional morphing, so this is no longer neccesary